ee/backend/app/src/main/kotlin/io/tolgee/ee/api/v2/controllers/slack/SlackSlashCommandController.kt

```diff
@@ -178,15 +181,18 @@ class SlackSlashCommandController(
         throw SlackErrorException(slackErrorProvider.getInvalidCommandError())
```

Comment:
I think, we should be more specific when throwing these errors. We should tell the user, exactly which event is incorrect. You can achieve this by creating separate function for getting `EventName` from the string and wrapping it with try-catch block.